### PR TITLE
Update version: RIA.DigiDoc4 version 4.4.0.4447

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117217)

### CODE REVIEW

File: manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+ReleaseDate: 2023-08-11
+AppsAndFeaturesEntries:
+- UpgradeCode: '{67932072-AA9E-45E3-B697-D825349F273D}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.4.0/Digidoc4_Client-4.4.0.4447.x64.en-US.qt.msi
+  InstallerSha256: 61525820CC07BA50DFE91EDB2701EF9F832A20E12D8518C46A0278F7F7BD21ED
+- Architecture: x64
+  InstallerUrl: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.4.0/Digidoc4_Client-4.4.0.4447.x64.en-US.qt.msi
+  InstallerSha256: 61525820CC07BA50DFE91EDB2701EF9F832A20E12D8518C46A0278F7F7BD21ED
```
Comment (CoolPlayLin) [inline manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml:18]:
```suggestion
```

File: manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+ProductCode: '{CA0193A7-FC2D-4DFD-87FB-AAF5FB4F2E8F}'
+ReleaseDate: 2023-08-11
+AppsAndFeaturesEntries:
+- UpgradeCode: '{67932072-AA9E-45E3-B697-D825349F273D}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.4.0/Digidoc4_Client-4.4.0.4447.x64.en-US.qt.msi
+  InstallerSha256: 61525820CC07BA50DFE91EDB2701EF9F832A20E12D8518C46A0278F7F7BD21ED
+- Architecture: x64
+  InstallerUrl: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.4.0/Digidoc4_Client-4.4.0.4447.x64.en-US.qt.msi
```
Comment (CoolPlayLin) [inline manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml:17]:
```suggestion
```

File: manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+InstallerType: wix
+ProductCode: '{CA0193A7-FC2D-4DFD-87FB-AAF5FB4F2E8F}'
+ReleaseDate: 2023-08-11
+AppsAndFeaturesEntries:
+- UpgradeCode: '{67932072-AA9E-45E3-B697-D825349F273D}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.4.0/Digidoc4_Client-4.4.0.4447.x64.en-US.qt.msi
+  InstallerSha256: 61525820CC07BA50DFE91EDB2701EF9F832A20E12D8518C46A0278F7F7BD21ED
+- Architecture: x64
```
Comment (CoolPlayLin) [inline manifests/r/RIA/DigiDoc4/4.4.0.4447/RIA.DigiDoc4.installer.yaml:16]:
```suggestion
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
>  Manifest Error: Duplicate installer entry found.

**Pull request closed without merging**